src/containers/Identity/Home/View.js

```diff
@@ -53,7 +60,9 @@ const Home = (props) => {
           {pinnedAttestations.keySeq().map((attestationKey) => (
             <TouchableOpacity
               key={pinnedAttestations.getIn([attestationKey, 'id'], '')}
-              onPress={() => goToAttestationDetails(pinnedAttestations.getIn([attestationKey, 'id'], ''), pinnedAttestations.getIn([attestationKey, 'claimName'], ''))}
+              onPress={() => goToAttestationDetails(pinnedAttestations.getIn([attestationKey, 'id'], ''),
+                pinnedAttestations.getIn([attestationKey, 'claimName'], ''),
+                pinnedAttestations.getIn([attestationKey, 'identityAttested'], ''))}
```

Comment:
```suggestion
              onPress={() => goToAttestationDetails(pinnedAttestations.get(attestationKey, IMap())}
```
Pass in the whole item.